engine/src/stackfileformat.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright (C) 2016 LiveCode Ltd.
+ 
+ This file is part of LiveCode.
+ 
+ LiveCode is free software; you can redistribute it and/or modify it under
+ the terms of the GNU General Public License v3 as published by the Free
+ Software Foundation.
+ 
+ LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+ WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ for more details.
+ 
+ You should have received a copy of the GNU General Public License
+ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+#ifndef __MC_STACKFILEFORMAT_H__
+#define __MC_STACKFILEFORMAT_H__
+
+#define kMCStackFileFormatVersion_1_0 (1000)
+#define kMCStackFileFormatVersion_1_3 (1300)
+#define kMCStackFileFormatVersion_1_4 (1400)
+#define kMCStackFileFormatVersion_2_0 (2000)
+#define kMCStackFileFormatVersion_2_3 (2300)
+#define kMCStackFileFormatVersion_2_4 (2400)
+#define kMCStackFileFormatVersion_2_7 (2700)
+#define kMCStackFileFormatVersion_5_0 (5000)
+#define kMCStackFileFormatVersion_5_5 (5500)
+#define kMCStackFileFormatVersion_7_0 (7000)
+#define kMCStackFileFormatVersion_8_0 (8000)
```

Comment:
It would be nice to make this into an enumeration.  The various places that work with stack file versions could then be adjusted (in a subsequent patch) to use a `MCStackFileFormatVersion` instead of a `uint4`.

The minimum/current values could then be `const MCStackFileFormatVersion` instead of macros.